raiden/tests/integration/test_send_queued_messages.py

```diff
@@ -165,25 +166,55 @@ def test_payment_statuses_are_restored(
 
     app0.event_handler = HoldRaidenEvent()
     app0.event_handler.hold(SendSecretReveal, {})
+    app2.event_handler = HoldRaidenEvent()
+    app2.event_handler.hold(SendSecretRequest, {})
+
+    # make one transfer from app0 to app2 so that the locked amount
+    # in the app0-app1 channel also includes collateral from other channels
+    identifier = 42
+    payment_status = app0.raiden.mediated_transfer_async(
```

Comment:
Is this necessary? I don't see this being used in the test.

Reply:
you mean `payment_status` as a variable or the extra transfer? The extra transfer is used to add additional locked amount into the app0-app1 channel.

Reply:
The extra transfer. How is this being used in the test?

This does indeed add a lock to the merkle tree, but there are already `spent_amount` locks in it, `locked_amount` is also equal to `spent_amount`, so in the context of the bug this is fixing the difference from `locked_amount` and `lock.amount` would be caught already.

Reply:
Yes you are correct. Then I will simplify the test. Makes it run faster :)